### DISCUSSION TITLE
Bad performance in Firefox?

### DISCUSSION THREAD

**Discussion by wonx** [EmulatorJS/EmulatorJS#907] [General]
I noticed that, while in Chrome the performance is ok, in Firefox the audio is very choppy and the framerate inconsistent. Seems to happen across systems. Has anyone experienced anything similar?

**Comment by ethanaobrien** [comment]
Yes, the Firefox web assembly runtime is slower than the chromium web assembly runtime

**Reply by wonx** [reply]
I see. Is there a way to tweak or optimize the settings in Firefox so we get similar performance?

**Reply by ethanaobrien** [reply]
There is not
